features/gift_exchanges/challenge_giftexchange.feature

```diff
@@ -348,6 +348,46 @@ Feature: Gift Exchange Challenge
     Then I should see "myname1"
       And I should see "Fulfilled Story"
 
+  Scenario: A mod can default all incomplete assignments
+    Given everyone has their assignments for "Awesome Gift Exchange"
+      And I am logged in as "myname1"
+      And I fulfill my assignment
+   When I am logged in as "mod1"
+     And I go to the "Awesome Gift Exchange" assignments page
+     And I follow "Default All Incomplete"
+   Then I should see "All unfulfilled assignments marked as defaulting."
+     And I should see "Undefault myname2"
+     And I should see "Undefault myname3"
+     And I should see "Undefault myname4"
+
+  Scenario: User can default and undefault on their assignment 
```

Comment:
The moderator, not the user, does the undefaulting, so I think the scenario here should be either "User can default on their assignment, and mod can undefault them."

It looks like you're _also_ testing that the mod can assign a pinch hitter to a user who has defaulted. Because that's unrelated to undefaulting a user, I'd suggest making that its own scenario, even if it does make the test take a tiny bit longer to run.

Also, the indenting for this scenario is off -- all the `Given`, `When`, and `Then` are indented 3 spaces, and all the `And` are indented 5.